### PR TITLE
Add CircleCI badge to README.

### PR DESCRIPTION
**Pull request by daspecster**

### CODE REVIEW

**Comment by daspecster** [comment]
Cancelled Travis.

File: README.rst [hunk 1]
```diff
@@ -115,6 +115,8 @@ Apache 2.0 - See `LICENSE`_ for more information.
 
 .. |build| image:: https://travis-ci.org/GoogleCloudPlatform/google-cloud-python.svg?branch=master
    :target: https://travis-ci.org/GoogleCloudPlatform/google-cloud-python
+.. |circleci| image:: https://circleci.com/gh/GoogleCloudPlatform/google-cloud-python.svg?style=shield&circle-token=:circle-token
```
Comment (dhermes) [inline README.rst]: Remove `circle-token=:circle-token`

Comment (daspecster) [inline README.rst]: https://github.com/GoogleCloudPlatform/google-cloud-python/pull/2790/commits/14ea041c88b4ff7f79bb7c494ceed08943ed4c56

**Comment by daspecster** [comment]
nit fixed! Squashed, and merging.

**Pull request merged**